web/cypress/integration/plugin/create-edit-delete-plugin.spec.js

```diff
@@ -63,16 +65,18 @@ context('Create and Delete Plugin List', () => {
 
   it('should delete plugin list', () => {
     cy.visit('/plugin/list');
-    cy.get(domSelector.refresh).click();
+    cy.get('.ant-btn-dangerous').should('exist');
+
     cy.get(domSelector.tableCell, { timeout }).should('exist').then(function (rows) {
       [...rows].forEach((row) => {
         const name = row.innerText;
-        const cases = this.cases[name] || [];
```

Comment:
We may have a fallback for those plugins that don't have valid or invalid data cases.

Reply:
you mean not all the plugins in the data-list will appear in the plugin list after the create and edit cases finished right? 

yap, It might make more sense to take the number of delete buttons as loop parameters. Let me try